components/support/crypto/nss-sys/build.rs

```diff
@@ -0,0 +1,120 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+use std::{
+    env,
+    ffi::OsString,
+    path::{Path, PathBuf},
+    process::{self, Command},
+};
+
+fn env(name: &str) -> Option<OsString> {
+    println!("cargo:rerun-if-env-changed={}", name);
+    env::var_os(name)
+}
+
+fn main() {
+    let (lib_dir, include_dir) = get_nss();
+    println!(
+        "cargo:rustc-link-search=native={}",
+        lib_dir.to_string_lossy()
+    );
+    println!("cargo:include={}", include_dir.to_string_lossy());
+}
+
+// Adapted from https://github.com/sfackler/rust-openssl/blob/546405dc58e0e11d695ba6957aab567f9d559192/openssl-sys/build/find_normal.rs
```

Comment:
I'm not sure about the legal nuances of adapting code from there (where AFAICT it's apache-licensed) and then re-licensing it under the MPL as part of this file. How much is the code the same between the two versions?

Reply:
Yup turns out we don't need all of this!